libraries/imageslider/src/main/java/com/trendyol/uicomponents/imageslider/ImageSliderView.kt

```diff
@@ -0,0 +1,63 @@
+package com.trendyol.uicomponents.imageslider
+
+import android.app.Activity
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import androidx.appcompat.app.AppCompatActivity
+import com.trendyol.uicomponents.imageslider.databinding.ViewImageSliderBinding
+import java.lang.ref.WeakReference
+
+class ImageSliderView @JvmOverloads constructor(
```

Comment:
Please avoid using JVM overloads constructor
https://medium.com/@mmlodawski/https-medium-com-mmlodawski-do-not-always-trust-jvmoverloads-5251f1ad2cfe